src/linear_algebra/clifford_algebra/equivs.lean

```diff
@@ -0,0 +1,89 @@
+/-
+Copyright (c) 2021 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import linear_algebra.clifford_algebra.basic
+import data.complex.module
+
+/-!
+# Other constructions isomorphic to Clifford Algebras
+
+This file contains isomorphisms showing that other types are equivalent to some `clifford_algebra`:
+
+* `clifford_algebra_ring.equiv`: any ring is equivalent to a `clifford_algebra` over an empty
+  vector space.
+* `clifford_algebra_complex.equiv`: the `complex` numbers are equivalent to a
+  `clifford_algebra` over a one-dimensional vector space with a quadratic form that satisfies
+  `Q (ι Q 1) = -1`.
+
+Future work:
+
+* Isomorphism to the `quaternion`s over `ℝ × ℝ`, sending `i, j, k` to `(0, 1)`, `(1, 0)`, and
+  `(1, 1)` (port from the `lean-ga` project).
```

Comment:
The context here is https://github.com/pygae/lean-ga/blob/master/src/geometric_algebra/from_mathlib/concrete/quaternion.lean, which I hope to PR at some point.